src/components/cart/CartProduct.jsx

```diff
@@ -0,0 +1,55 @@
+import "./cart-product.css";
+import { useCart } from "../../context/cart-context";
+const CartProduct = ({ product }) => {
+
+    const { addToCart, removeFromCart, decreaseProductCountInCart } = useCart();
+
+    function getDiscountOfProduct(price, originalPrice) {
+        //discPercent = originalPrice - sellingPrice / originalPrice * 100
+        return parseInt(((Number(originalPrice) - Number(price)) / Number(originalPrice)) * 100)
+    }
+
+    let discount = getDiscountOfProduct(product.price, product.originalPrice)
+
+    return (
```

Comment:
you can destructure the product at the above like {price,originalPrice,photo,title}=product